src/engine/client/client.cpp

```diff
@@ -954,9 +992,11 @@ int CClient::UnpackServerInfo(CUnpacker *pUnpacker, CServerInfo *pInfo, int *pTo
 			NumPlayers++;
 	}
 	pInfo->m_NumPlayers = NumPlayers;
```

Comment:
`NumPlayers` is misused. Maybe remove this line ?

Reply:
Ah, yep I missed that one